app/src/main/res/drawable/shadow.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What are you using this xml for?
@codestaintin

Reply:
This was used to add a shadow to the card image.

Reply:
And the image appears that way because I used the emulator's screenshot button to take the screenshot.
@TheDancerCodes

Reply:
What do you mean by extra edge? Is this being done at runtime? Do you mean that on a real device the image spans the whole width?
@codestaintin